service.onmute/service.py

```diff
@@ -0,0 +1,215 @@
+#   Copyright (C) 2017 Lunatixz
+#
+#
+# This file is part of OnMute
+#
+# OnMute is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# OnMute is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with OnMute.  If not, see <http://www.gnu.org/licenses/>.
+
+import sys, os, re, traceback, json
+import xbmc, xbmcplugin, xbmcaddon, xbmcgui
+
+# Plugin Info
+ADDON_ID = 'service.onmute'
+REAL_SETTINGS = xbmcaddon.Addon(id=ADDON_ID)
```

Comment:
Na, not a requirement.. Just trying to get your habit away :P (No change needed unless you want to)